Source/Scene/Scene.js

```diff
@@ -2696,6 +2706,68 @@ define([
         return object;
     };
 
+    var scratchPickDepthPassState;
+
+    function renderForPickDepth(scene, drawingBufferPosition) {
```

Comment:
Perhaps add a `PERFORMANCE_IDEA` about rendering translucent only and merging with the previous frame; I still think it could work if it checks the state is in sync.